OpenRA.Mods.Common/Traits/AttackMove.cs

```diff
@@ -121,9 +122,13 @@ protected virtual IEnumerable<Order> OrderInner(World world, CPos cell, MouseInp
 			}
 		}
 
-		public override void Tick(World world)
+		public override void SelectionChanged(World world, IEnumerable<Actor> selected)
 		{
-			if (subjects.All(s => s.Actor.IsDead))
+			subjects = selected.SelectMany(a => a.TraitsImplementing<AttackMove>()
+					.Select(am => new TraitPair<AttackMove>(a, am)))
+				.ToArray();
+
+			if (!subjects.Any(s => s.Actor.Info.HasTraitInfo<AutoTargetInfo>()))
```

Comment:
Assigning subjects based on `AttackMove` and then checking `AutoTargetInfo` here looks like a typo. If this is intentional, can we have a comment explaining that?

Reply:
See above.